video/cloud-client/analyze/video_detect_faces_gcs_beta_test.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import video_detect_faces_gcs_beta
+
+RESOURCES = os.path.join(os.path.dirname(__file__), "resources")
+
+
+def test_detect_faces(capsys):
+    input_uri = "gs://cloud-samples-data/video/googlework_short.mp4"
```

Comment:
googlework_tiny is used in the tests of person_detection.  perhaps use it here too?

Reply:
The googlework_tiny video doesn't register any faces so unfortunately the test fails. I had to use this one. I could instead use this one in person_detection?